### ISSUE TITLE
Deleting passwords scrolls to the top

### ISSUE THREAD

**Issue by summersab** [marius-wieschollek/passwords#338]
**Current Status**
Deleting a password causes the page to scroll to the top. This makes cleaning up duplicate passwords/accounts very difficult.

**Feature Description**
The row should be deleted without interrupting the current scroll position.

**Additional context**
I wasn't sure if this should be a feature request or a bug, but nothing is broken, so I put it here.

**Comment by ozls** [comment]
I totally agree. It does the same when you add a password to your favorites or you edit it. That being said it sounds like the same issue as #287, dating back to August 2020. Is there anything being done ? It makes it very difficult to organize and update passwords, which in turn defeats the efforts made until now to make people use secure passwords.

**Comment by politas** [comment]
Yeah, I think it's any save action that does it.
